Samples/src/SimpleGrover/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Quantum.Simulation.Simulators;
+using System;
+
+namespace Microsoft.Quantum.Samples.SimpleGrover
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var sim = new QuantumSimulator();
+            var nDatabaseQubits = 5;
+
+            var result = ApplyGrover.Run(sim, nDatabaseQubits).Result;
+            Console.WriteLine($"Result: {result}");
+
+            System.Console.WriteLine("\n\nPress any key to continue...\n\n");
+            System.Console.ReadKey();
```

Comment:
For consistency with line 17, I suggest to remove `System.` from the beginning of the line.

Reply:
Good catch, thanks!